### PR TITLE
Update assign-licenses-to-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about group-based licensing in case users are trying to assign licenses to thousands of users.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review.

Please note that I was not able to use the relative link and had to use the absolute link. Could this be because it was from Azure repo?

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4822

File: microsoft-365/admin/manage/assign-licenses-to-users.md [hunk 1]
```diff
@@ -124,6 +124,10 @@ When you use the **Active users** page to assign licenses, you assign users lice
     By default, all services associated with those licenses are automatically assigned to the users. You can limit which services are available to the users. Switch the toggles to the **Off** position for the services that you don't want the users to have.
 6. At the bottom of the pane, select **Add** \> **Close**.  
 
+
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/manage/assign-licenses-to-users.md:128]: Replace with the following:

`If you want to assign licenses for a large number of users, use [Assign licenses to users by group membership in Azure Active Directory](/azure/active-directory/enterprise-users/licensing-groups-assign).`

Also, in the "Before you begin" section, please replace the second bullet with the following:

`You can [assign Microsoft 365 licenses to user accounts with PowerShell](../../enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell.md).`

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please make the 2 changes as noted.

**Review by JohanFreelancer9 — COMMENTED** [review]
@msbemba - I'll do a copy review once @cmcatee-MSFT's suggested changes have been implemented. Thanks.

File: microsoft-365/admin/manage/assign-licenses-to-users.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/admin/manage/assign-licenses-to-users.md:128]: Thank you, Cern. Made changes as per your request.

Thanks
Sri

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT

**Pull request merged**